docs/Developers/Review.rst

```diff
@@ -75,13 +77,23 @@ The review process for coala is as follows:
 
     .. note::
 
-        Only one acknowledgment is needed per commit i.e ``ack commit_SHA``.
+          Only one acknowledgment is needed per commit i.e ``ack commit_SHA``.
```

Comment:
Why have you added leading white spaces here ?

Reply:
This change here is not required IMO

Reply:
Should it be mentioned after members review points?

Reply:
Yes that would be better

Reply:
okay! 👍